content/applications/finance/fiscal_localizations/germany.rst

```diff
@@ -330,3 +196,135 @@ What happens if you are not compliant?
+   Sale** app (`point_of_sale`) and the **Restaurant** module (`pos_restaurant`).
+#. :ref:`Install <general/install>` the **Germany - Certification for Point of Sale**
+   (`l10n_de_pos_cert`) and **Germany - Certification for Point of Sale of type restaurant**
+   (`l10n_de_pos_res_cert`) modules.
+
+   .. tip::
+      If these modules are not listed, :ref:`update the app list <general/install>`.
+
+.. image:: germany/pos-upgrade.png
+   :align: center
+   :alt: Upgrading Odoo Point of Sale from the Apps dashboard
+
+Register your company at the financial authority
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To register your company, go to :menuselection:`Settings --> General Settings --> Companies -->
+Update Info`, fill out the following fields and *Save*.
+
+- **Company name**
+- Valid **address**
+- **VAT** number
+- **St.-Nr** (Steuernummer): this number is assigned by the tax office to every taxable natural or
+  legal person. (e.g., `2893081508152`)
+- **W-IdNr** (Wirtschafts-Identifikationsnummer): this number is used as a permanent
+  identification number for economically active persons.
+
+You can then **register your company through fiskaly** by opening the *fiskaly* tab and clicking on
+the *fiskaly Registration* button.
+
+.. image:: germany/fiskaly-registration.png
+   :align: center
+   :alt: Button to register a company through fiskaly in Odoo
+
+.. tip::
+   If you do not see the *fiskaly Registration* button, make sure that you *saved* your company
+   details and are not in *editing mode* anymore.
+
+Once the registration has been finalized, new fields appear:
+
+- **fiskaly organization ID** refers to the ID of your company at the fiskaly side.
+- **fiskaly API key** and **secret** are the credentials the system uses to access the services
+  offered by fiskaly.
+
+.. image:: germany/fiskaly-keys.png
+   :align: center
+   :alt: fiskaly keys as displayed on Odoo
+
+.. note::
+   It is possible to request new credentials if there is any issue with the current ones.
+
+Create and link a Technical Security System to your PoS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: germany/create-tss.png
+   :align: right
+   :alt: Create TSS option from a point of sale
+
+To use your point of sale in Germany, you first have to create a :abbr:`TSS (Technical Security
+System)` for it.
+
+To do so, go to :menuselection:`Point of Sale --> Configuration --> Point of Sale`, open the point
+of sale you want to edit, then check the box next to **Create TSS** and *Save*.
+
+.. image:: germany/tss-ids.png
+   :align: right
+   :alt: Example of TSS ID and Client ID from fiskaly in Odoo Point of Sale
+
+Once the creation of the TSS is successful, you can find your **TSS ID** and **Client ID** under the
+*fiskaly API* section.
+
+- **TSS ID** refers to the ID of your TSS at fiskaly's side.
+- **Client ID** refers to your PoS but at fiskaly's side.
```

Comment:
Once the TSS has been created, fill in the fields:

- :guilabel:`TSS ID`: refers to the ID of your TSS on fiskaly.
- :guilabel:`Client ID`: refers to your PoS on fiskaly.